support-frontend/assets/helpers/abTests/abtestDefinitions.ts

```diff
@@ -100,4 +101,29 @@ export const tests: Tests = {
 		targetPage: pageUrlRegexes.contributions.allLandingPagesAndThankyouPages,
 		seed: 1,
 	},
+	singleLessProminent: {
+		variants: [
+			{
+				id: 'control',
+			},
+			{
+				id: 'variant',
+			},
+		],
+		audiences: {
+			ALL: {
+				offset: 0,
+				size: 1,
+			},
+		},
+		isActive: true,
+		referrerControlled: false,
+		targetPage: pageUrlRegexes.contributions.allLandingPagesAndThankyouPages,
+		seed: 11,
+		canRun: () => {
+			// If URL query string parameter contains annual or monthly contributionType apply abTest
```

Comment:
The comment isn't quite right, If `contributionTypesFromUrl` is not ONE_OFF we apply the abTest, so it could be `null` as well as "annual" or "monthly" and we'd apply the abTest.

Reply:
Ok, comments should follow the logic, done.